go/kubebuilder/memcached-operator/api/v1alpha1/memcached_webhook.go

```diff
@@ -71,7 +74,6 @@ func (r *Memcached) ValidateDelete() error {
 
 	return nil
 }
-
 func validateOdd(n int32) error {
```

Comment:
TODO: There may be formatting required after the script is run. Will `go fmt` to this script and check this out.